### DISCUSSION TITLE
Global search (space + /) with the yanked value?

### DISCUSSION THREAD

**Discussion by theharshpat** [helix-editor/helix#8268] [Q&A] [answered]
Let's say I yank "abc"

1) as it is
How to paste the yanked value as (space + /) input value. How do I simply search for what is in the register?

2) In combination with other characters
Example use case in typescript:
interface abc;

To find it as an only return value, I will have to global search "): abc<T> {" or ": abc"

**Comment by rokkerruslan** [comment]
1. Try this: yank text, `*`, `Space + /`, `Enter`. Doc - https://docs.helix-editor.com/keymap.html#search `* | Use current selection as the search pattern | search_selection`

**Reply by theharshpat** [reply]
Thanks, that helps. But it seems we cant edit that string. So can't handle case 2.

**Comment by lunjon** [comment, marked as answer]
I believe you can use registers in global search (also in buffer search). So just ctrl-r + ", for instance, to add the default register content when searching.

**Reply by theharshpat** [reply]
Thanks, that's really useful
